src/pyflex/window_selector.py

```diff
@@ -306,6 +321,14 @@ def calculate_preliminiaries(self):
             self.peaks = self.peaks[(self.peaks > min_trough) &
                                     (self.peaks < max_trough)]
 
+    def print_remaining_windows(self):
```

Comment:
Do you think it makes sense to put this function in window.py?

Since all it only requires window object.